### ISSUE TITLE
Please add .env file as well

### ISSUE THREAD

**Issue by deepanshurathi553** [ONDC-Official/biap-client-node-js#6]

**Comment by rohanrajpal** [comment]
Yeah a `.sample.env` would be appreciated and probably a method to procure the sanbox values for the .env file

**Comment by ashok-murugan** [comment]
for now we can refer readme file in https://github.com/ONDC-Official/ondc-sdk page, env config provided here
